libs/theme/src/variables.css

```
@@ -19,4 +19,55 @@
 	--background-color: white;
 	--background-contrast-color: black;
 	--error-color: red;
+
+	/* Colors */
+	--surface-color: rgb(255 255 255 / 100%);
+	--surface-overlay-color: rgb(0 0 0 / 5%);
+	--on-surface-color: rgb(0 0 0 / 100%);
+	--on-surface-dim-color: rgb(0 0 0 / 50%);
+	--on-primary-color: rgb(255 255 255);
+	--border-color: rgb(0 0 0 / 15%);
+	--focus-outline-color: #212121;
+
+	/* Spacing */
+	--spacing-sm: 8px;
+	--spacing-md: 16px;
+	--spacing-default: 20px;
+	--spacing-lg: 24px;
+	--spacing-xl: 32px;
+	--spacing-xxl: 40px;
```

It's better to use base spacing + 'calc' function. It may look like this:
```suggestion
	--spacing-sm: 8px;
	--spacing-md: calc(var(--spacing-sm) * 2);
	--spacing-default: calc(var(--spacing-sm) * 2.5);
	--spacing-lg: calc(var(--spacing-sm) * 3);
	--spacing-xl: calc(var(--spacing-sm) * 4);
	--spacing-xxl: calc(var(--spacing-sm) * 5);
```